docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+# This file can be used to create a development environment for Wordpress
+# using Docker.
+#
+#
+# To set up
+# =========
+#
+# Download a recent backup of the database from Azure:
+#
+#     Azure > 
+#     Resource groups > 
+#     bibletranslationtools (Resource Group) > 
+#     bibletranslationtools (App Service) > 
+#     Backups > 
+#     (click backup) > 
+#     Download
+#
+# Unzip the file into $TEMP/bibletranslationtools.  There should now be a
+# file called bibletranslationtools.sql in the directory.
+#
+# Edit bibletranslationtools.sql and add the following lines at the top:
+#
+#     CREATE DATABASE bibletranslationtools;

```

Comment:
Would be nice to see that as an actual script modification with bash instead of a manual step.

Reply:
I agree, it would be nice to save a manual step.  I don't know if I can invoke a bash script during Docker composition to massage the file before the MySQL initialization sees it.